cmd/snap/cmd_services.go

```diff
@@ -184,8 +185,57 @@ func (s *svcLogs) Execute(args []string) error {
 	return nil
 }
 
+type userAndScopeMixin struct {
+	System bool   `long:"system"`
+	User   bool   `long:"user"`
+	Users  string `long:"users" optional:"yes" optional-value:"*"`
+}
+
+var userAndScopeDescs = mixinDescs{
+	// TRANSLATORS: This should not start with a lowercase letter.
+	"system": i18n.G("The operation should only affect system services."),
+	// TRANSLATORS: This should not start with a lowercase letter.
+	"user": i18n.G("The operation should only affect user services for the current user."),
+	// TRANSLATORS: This should not start with a lowercase letter.
+	"users": i18n.G("The operation should affect all user services."),
```

Comment:
still feels a bit odd, maybe @degville has some ideas how to phrase that nicely

```suggestion
	"users": i18n.G("The operation should affect user services of all or selected, comma separated, list of users."),
```

Reply:
I think pebble now uses `--users=all` on the notices command, we should follow that. And also teach the other snap commands that take --users  to understand --users=all which for them might correspond to the default behavior. We don't need to support username here for now.

Reply:
I have left the description as-is